File: Sirma Academy May 2024/JavaScript/Conditional Statements/Speed.js
function speedCalc(speed) {
  if (speed <= 10) {
    console.log("slow");
  } else if (speed <= 60) {
    console.log("average");
  } else if (speed <= 120) {
    console.log("fast");
  } else if (speed <= 160) {
    console.log("super-fast");
  } else {
    console.log("turbo-fast");
  }
}
speedCalc(2);

File: Sirma Academy May 2024/Java/Conditional Statements/src/Speed.java
import java.util.Scanner;

public class Speed {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double speed = Double.parseDouble(sc.nextLine());

        if (speed <= 10) {
            System.out.println("slow");
        } else if (speed <= 60) {
            System.out.println("average");
        } else if (speed <= 120) {
            System.out.println("fast");
        } else if (speed <= 160) {
            System.out.println("super-fast");
        } else {
            System.out.println("turbo-fast");
        }
    }
}
